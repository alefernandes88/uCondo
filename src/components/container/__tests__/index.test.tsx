import React from 'react'
import { Text } from 'react-native'
import { render } from '@testing-library/react-native'
import Container from '../index'

describe('Components -> Container', () => {
  it('should render with correct style', () => {
    const { toJSON } = render((
        <Container>
          <Text>
            Any Children
          </Text>
        </Container>
    ))

    expect(toJSON()).toMatchSnapshot()
  })
})
