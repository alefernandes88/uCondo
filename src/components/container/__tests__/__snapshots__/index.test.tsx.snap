// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components -> Container should render with correct style 1`] = `
<View
  style={
    {
      "backgroundColor": "#F0EDF5",
      "borderBottomLeftRadius": 0,
      "borderBottomRightRadius": 0,
      "borderTopLeftRadius": 24,
      "borderTopRightRadius": 24,
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
      "paddingBottom": 18,
      "paddingLeft": 22,
      "paddingRight": 22,
      "paddingTop": 18,
    }
  }
>
  <Text>
    Any Children
  </Text>
</View>
`;
